version_helpers.py

```diff
@@ -1,7 +1,5 @@
 """ Distutils / setuptools helpers for versioning
 
-This code started life in the nibabel package as nisexts/sexts.py
```

Comment:
I'd suggest leaving this comment in, maybe adding a comment that it has been deprecated upstream in nibabel.

Reply:
This file will be removed before the end of the summer (september).  This is a way to start the process 😄